pyproject.toml

```diff
@@ -131,6 +131,7 @@ dev = [
 ]
 dev_all = [
     "tox",
+    "tox-uv>=1.11.3",
```

Comment:
Doesn't `tox-uv` depend on `tox`, i.e., can we remove the line above?

Reply:
yes, tox-uv depends on tox, but I'd rather keep all direct dependencies explicit: doing otherwise defeats the purpose of `--resolution=lowest-direct` (even though in this specific case it doesn't really matter). Instead, I'd like to actually give some lower bound to what version of tox we need.